public/locales/en-US/grafana.json

```diff
@@ -13,7 +13,7 @@
       "search": "Search"
     },
     "search-box": {
-      "placeholder": "Search Grafana"
+      "placeholder": "Search or jump to..."
```

Comment:
do you think we'll have any issues with this being different to the top search bar placeholder now, or should we try to keep them the same? 🤔

Reply:
@ashharrison90 I was thinking we change the placeholder text of global search input to match